ext/hal/Kconfig

```diff
@@ -14,6 +14,8 @@ source "ext/hal/atmel/asf/Kconfig"
 
 source "ext/hal/altera/Kconfig"
 
+source "ext/hal/cec/Kconfig"
```

Comment:
Let's use "ext/hal/microchip/cec" instead

Reply:
I make it cec1702 then. There's also cec1302 which is older version. Has same cpu but different peripheral blocks (especially the crypto side).